sql/dolphinscheduler-postgre.sql

```diff
@@ -769,8 +769,7 @@ VALUES ('admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', '', '0',
 
 -- Records of t_ds_alertgroup，dolphinscheduler warning group
 INSERT INTO t_ds_alertgroup(id,alert_instance_ids, create_user_id, group_name, description, create_time, update_time)
-VALUES (1, '1,2','dolphinscheduler warning group', 'dolphinscheduler warning group', '2018-11-29 10:20:39',
-        '2018-11-29 10:20:39');
+VALUES (1,'1,2', 1, 'default admin warning group', 'default admin warning group', '2018-11-29 10:20:39', '2018-11-29 10:20:39');
```

Comment:
Please replace `dolphinscheduler warning group` with `default admin warning group` globally. There should be four places that need to be modified including comments.

Reply:
> Please replace `dolphinscheduler warning group` with `default admin warning group` globally. There should be four places that need to be modified including comments.

Considering this is  yakcy's first time for contributing, replacing the comments could submit another PR